x-pack/plugins/apm/public/components/shared/charts/instances_latency_distribution_chart/custom_tooltip.stories.tsx

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TooltipInfo } from '@elastic/charts';
+import React, { ComponentType } from 'react';
+import { EuiThemeProvider } from '../../../../../../../../src/plugins/kibana_react/common';
+import { getDurationFormatter } from '../../../../../common/utils/formatters';
+import { PrimaryStatsServiceInstanceItem } from '../../../app/service_overview/service_overview_instances_chart_and_table';
+import { CustomTooltip } from './custom_tooltip';
+
+function getLatencyFormatter(props: TooltipInfo) {
```

Comment:
We should ask the elastic charts team to make `TooltipInfo` a generic so we can get better typing of `datum`:

```suggestion
function getLatencyFormatter(props: TooltipInfo<PrimaryStatsServiceInstanceItem>) {
```